# Python Exceptions using try except and finally